crates/soa/src/lib.rs

```diff
@@ -1,5 +1,7 @@
+mod either_index;
```

Comment:
If the goal is for `soa` to be `#![no_std]`, can we mark it as such to maintain that contract?